agora/footfall_ai/src/main.py

```diff
@@ -0,0 +1,141 @@
+import os
+import time
+import collections
+import cv2
+import numpy as np
+import torch
+from flask import Flask, render_template, Response, request
+from ultralytics import YOLO
+from ultralytics.solutions import ObjectCounter
+from video_capture import VideoCapture
+
+# Constants
+MODEL_PATH = "C:\\Users\\fcabrera\\Downloads\\jumpstart-apps\\agora\\footfall_ai\\src\\models\\yolov8n.pt"
```

Comment:
I think this has to point to the relative path of the model files in the repo?

Reply:
Yes, this is a mistake - Models shouldnt be part of github, and once in kubernetes, we will get the model path from an ENV variable.

Reply:
Fixed